### ISSUE TITLE
name backlisting

### ISSUE THREAD

**Issue by Guiorgy** [BaronKiko/LauncherHijack#49]
We all know that Amazon blacklisted this app's name. I have a feeling that even if the project name is changes, it will get blacklisted again. So I think the smartest will be be to include a quick tutorial to change application name in the readme. Let every one choose the name for themselves.

**Comment by ShayBox** [comment]
no, the best thing would be to have a random app name, like gameguardian

**Comment by Guiorgy** [comment]
I mean, if this was reuploaded as that, it would just be added into the blacklist... Every time BaronKiko changes it it will be blacklisted.

**Comment by ShayBox** [comment]
no I mean randomly generated like the game guardian app does, completely random

**Comment by ShayBox** [comment]
either a build server changing the name on interval, every x downloads, etc, or what game guardian does, reinstall itself with a random name

**Comment by Guiorgy** [comment]
you would need 'something' to rebuild the apk, be it a dedicated server or a 3rd party app. A simpler yet effective solution is to __*at least*__ provide a quick tutorial (with screenshots) to the users on how to do it manually

**Comment by ShayBox** [comment]
yeah but requiring users to install Android studio and everything to change one variable is really not gonna happen, people will just end up sharing apks, like I said, either a build server that builds it automatically with a random name, or the app can reinstall itself with a random name like game guardian,  both are acceptable but I would go with the server as it's easier, and build servers can be setup for free

**Comment by Guiorgy** [comment]
I just found out, but on February this year, an exploit was found on the [xda forum](https://forum.xda-developers.com/hd8-hd10/orig-development/experimental-software-root-hd-8-hd-10-t3904595). It only grants you root temporarily (while screen is on), but it is enough to run `pm disable com.amazon.firelauncher` and dissable default launcher. So far I have uninstalled LauncherHijack and Nova launcher works just fine. It is prety easy to use with just 6 small comands to run.

**Comment by ShayBox** [comment]
There's perm root for the HD 8 2018, I have it

**Comment by Guiorgy** [comment]
> There's perm root for the HD 8 2018, I have it

If you mean the exploit where you short a pice on the back of the phone to install a custom recovery, then it will require a factory reset and downgrade, not to mention that theres plenty of room to brick your device, and it is an overkill if all you want is to replace the launcher!

**Comment by ShayBox** [comment]
No, the software one

**Comment by Guiorgy** [comment]
> No, the software one

a link would be appreciated. ^_^

**Comment by ShayBox** [comment]
[Here](https://lmgtfy.com/?q=hd+8+2018+root) Second link
